### PR TITLE
Multiple fixes

### PR DESCRIPTION
**Pull request by Ricardo-Osorio**
Replace a third party lib method we used to parse the input from `--endpoints`. With the changes we will both have a shorthand `-e` and also accept the input in a much more user friendly way like `-e "10.71.26.0:5705,10.71.27.116:5705"` or  `-e "10.71.26.0:5705, 10.71.27.116:5705"`
To that I also added an example to the flag that shows a clear example: 
```
-e, --endpoints strings   set the list of endpoints used when connecting to the StorageOS API in the form of "10.71.26.0:5705, 10.71.27.116:5705" (default [http://localhost:5705])
```

Fix a bug with the filtering of volumes based on their healths (if statement was incorrectly enabling the filtering)

Run go mod tidy

image

### CODE REVIEW

File: config/flags/flags.go [hunk 1]
```diff
@@ -250,10 +251,11 @@ func NewProvider(flagset FlagSet, fallback config.Provider) *Provider {
 		config.DefaultAuthCacheDisabled,
 		"disable the CLI's caching of authentication sessions",
 	)
-	flagset.StringArray(
```
Comment (Mojachieee) [inline config/flags/flags.go:253]: Does this affect the way users currently input a list of endpoints? 

I'm happy adding new ways, but I don't want to break the existing workflow at the same time

Comment (Ricardo-Osorio) [reply config/flags/flags.go:253]: I don't think the chance of someone currently using scripts/automation together with this CLI tool is high enough to make it worth the added complexity of implementing two inputs for the same thing and logic to merge both

Comment (Mojachieee) [reply config/flags/flags.go:253]: Let's ask Ferran, as I know some customers have automation around us

**Review by Mojachieee — APPROVED** [review]
Discussed offline, LGTM

**Pull request merged**